src/pages/permissions/panels/EditGroupPermissionsPanel.tsx

```diff
@@ -0,0 +1,267 @@
+        panelParams.clear();
+      })
+      .catch((e) => {
+        notify.failure("Failed to update permissions", e);
+      })
+      .finally(() => {
+        setSubmitting(false);
+      });
+  };
+
+  const filteredPermissions = search
+    ? permissions.filter((permission) => {
+        return (
+          permission.entitlement.includes(search) ||
+          permission.entity_type.includes(search) ||
+          permission.url.includes(search)
+        );
+      })
+    : permissions;
+
+  const headers = [
+    { content: "Resource type", sortKey: "resourceType" },
+    { content: "Resource", sortKey: "resource" },
+    { content: "Entitlement", sortKey: "entitlement" },
+    { "aria-label": "Actions", className: "u-align--right actions" },
+  ];
+
+  const rows = filteredPermissions.map((permission, idx) => {
+    const resourceName = extractResourceNameFromUrl(
+      permission.entity_type,
+      permission.url,
+    );
+
+    const permissionId = getPermissionId(permission);
+
+    return {
+      name: `permission-${idx}`,
+      className: "u-row",
+      columns: [
+        {
+          content: permission.entity_type,
+          role: "cell",
+          "aria-label": "Resource type",
+        },
+        {
+          content: resourceName,
+          role: "cell",
+          "aria-label": "Resource",
+          className: "u-truncate",
+          title: resourceName,
+        },
+        {
+          content: permission.entitlement,
+          role: "cell",
+          "aria-label": "Entitlement",
+          className: "u-truncate",
+          title: permission.entitlement,
+        },
+        {
+          className: "actions u-align--right",
+          content: (
+            <Button
+              appearance="base"
+              hasIcon
+              dense
+              onClick={() => handleDeletePermission(permissionId)}
+              type="button"
+              aria-label="Delete permission"
+              title="Delete permission"
+            >
+              <Icon name="delete" />
+            </Button>
+          ),
+          role: "cell",
+          "aria-label": "Delete permission",
+        },
+      ],
+      sortData: {
+        resourceType: permission.entity_type,
+        resource: resourceName.toLowerCase(),
+        entitlement: permission.entitlement,
+      },
+    };
+  });
+
+  return (
+    <>
+      <SidePanel
+        isOverlay
+        loading={false}
+        hasError={false}
+        className="edit-permissions-panel"
+      >
+        <SidePanel.Header>
+          <SidePanel.HeaderTitle>{`Change permissions for ${group?.name}`}</SidePanel.HeaderTitle>
+        </SidePanel.Header>
+        <SidePanel.Content className="u-no-padding">
+          <NotificationRow className="panel-notification" />
+          <Card title="Add permissions">
+            <span>
+              In fine-grained authorisation entitlements need to be given in
```

Comment:
```suggestion
              Entitlements need to be given in
```
I think shortening the copy makes it easier to read?

Reply:
Yes agreed. I also have a [request](https://www.figma.com/file/0IsMmV1rsa6flOo6h4CjAV?type=design&node-id=1166-133202&mode=design#777963729) to update the copy for this panel overall. In my opinion the wording here isn't correct for lxd.